<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="toolButtonVideoPath">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>118</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineH">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>55</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineVideoPath">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>117</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonXML">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>202</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select detection area</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelW">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>380</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonXMLfile">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>146</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineW">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>55</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineImagePath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxWebcam">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Webcam 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Webcam 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Webcam 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Webcam 4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelH">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRectangle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw rectangle around object in the video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImagePath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>276</y>
     <width>86</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineXMLfile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>146</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideoPath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>122</y>
     <width>86</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Path:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxsaveImages">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>251</y>
     <width>86</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Save images</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelWebcam">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Webcam:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelXMLfile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>148</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Area File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResolution">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Resolution:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonImagePath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>179</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Area Status:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineStatus">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>177</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelMinPos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>303</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum required number of positive detections:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMinPosRequired">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodec">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>86</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Codec:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCodec">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>83</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Raw Video</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FFV1 Lossless Video</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lagarith Lossless Video</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>85</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>UFOID User Token:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineToken">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>326</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonToken">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>326</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
